integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -746,19 +744,19 @@ object BloopDefaults {
 
       import scala.collection.JavaConverters._
       val visited = Classpaths.interSort(currentProject, conf, data, deps)
-      val productDirs = (new java.util.LinkedHashSet[Task[Seq[File]]]).asScala
-      val bloopProductDirs = (new java.util.LinkedHashSet[Task[Seq[File]]]).asScala
+      val productDirs = new java.util.LinkedHashSet[Task[Seq[File]]].asScala
```

Comment:
This is more difficult to parse, revert?

Reply:
Okay. IntelliJ was pointing it out too.